broker/consumer/vm_startup.sh

```diff
@@ -87,17 +87,28 @@ do
     # which kills the while loop. no working suggestions found.
     # passing the error with `|| true`
 
+    
+    IFS=', ' read -r -a KAFKA_TOPIC_ARRAY <<< "$KAFKA_TOPIC"
+    KAFKA_TOPIC_PRESENT=()
+
     # check if our topic is in the list
-    if grep -Fq "${KAFKA_TOPIC}" "${fout_topics}"
-    then
-        alerts_flowing="true"  # start consuming
+    for topic in "${KAFKA_TOPIC_ARRAY[@]}"; 
+    do
+        if grep -Fq "${topic}" "${fout_topics}"
+        then
+            KAFKA_TOPIC_PRESENT+=("True")
+        fi
+    done
+    
+    if [ ${#KAFKA_TOPIC_PRESENT[@]} -eq ${#KAFKA_TOPIC_ARRAY[@]} ]; then
+        alerts_flowing="true"
```

Comment:
Add a comment to the first line explaining what it does. I imagine it's something like, "check whether the two arrays are the same length".

If I understand correctly, this will cause the Connector to start up only if all topics are present. If so:
1. This should be explicitly stated in the comment above the code block (line 94).
2. In #212 you said:

    > Even if one of the (multiple) topics is not live, the consumer will ingest alerts from the topics that are live.

    I'm guessing that what you actually tested was the behavior of the _Kafka -> Pub/Sub Connector_ (launched by the last command in the script `/bin/connect-standalone ...`). I recommend updating the languaging in the issue to reflect that.

Reply:
>    I'm guessing that what you actually tested was the behavior of the _Kafka -> Pub/Sub Connector_ (launched by the last command in the script `/bin/connect-standalone ...`). I recommend updating the languaging in the issue to reflect that.

Yes, this is correct. I will update the main comment shortly